app/redux/types/questTypes.ts

```diff
@@ -11,3 +11,87 @@ export interface ISaveCurrentQuestLineQuests {
 }
 
 export type IDailyCheck = Record<string, string>;
+
+export interface IInterruptedQuestLine {
+  id: string;
+  day: number;
+  interruptedQuestInx: number;
+}
+
+export const ALL_QUESTS_STACK = [
```

Comment:
it isn't stack
it's array

name it as `ALL_QUEST_IDS_BY_DAYS`

also it would be useful if we name if not only with comments
something like this:
```
[
[{ id: 5560822, name: 'Healing Hand - The Charm of Healing' }, { id: 6341245, name: 'BlaBlaBla' }],
[ ... ],
[... ],
...
]
```